lm_eval/evaluator.py

```diff
@@ -592,7 +592,7 @@ def evaluate(
             "n-samples": {
                 task_output.task_name: {
                     "original": len(task_output.task.eval_docs),
-                    "effective": min(limit, len(task_output.task.eval_docs)),
+                    "effective": None if limit is None else min(limit, len(task_output.task.eval_docs)),
```

Comment:
I think if `limit` is `None` then `effective` should be `len(task_output.task.eval_docs)`

Reply:
thanks, fixed